### PR TITLE
fix content_template library_id reference in delete

### PR DESCRIPTION
**Pull request by mikemorency**
##### SUMMARY
Fixes https://github.com/ansible-collections/vmware.vmware/issues/90

While refactoring the content_template module, one variable reference was missed. There are no tests for the spot that this issue occurred. I fixed the variable and added tests to cover this spot.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
content_template

### CODE REVIEW

File: tests/integration/targets/vmware_content_template/tasks/main.yml [hunk 1]
```diff
@@ -60,6 +60,20 @@
...
+        username: "{{ vcenter_username }}"
+        password: "{{ vcenter_password }}"
+        port: "{{ vcenter_port }}"
+        template: "{{ template_name }}"
+        library: "{{ library }}"
+        vm_name: "{{ vm }}"
+        host: "{{ template_host }}"
+        state: absent
+      when: not run_on_simulator
+
```
Comment (anna-savina) [inline tests/integration/targets/vmware_content_template/tasks/main.yml:76]: Maybe also add verification that the template was deleted. Something like this.
```suggestion
    - name: Verify template deleted
      vmware.vmware.content_library_item_info:
        validate_certs: false
        hostname: "{{ vcenter_hostname }}"
        username: "{{ vcenter_username }}"
        password: "{{ vcenter_password }}"
        port: "{{ vcenter_port }}"
        library_name: "{{ library }}"
        library_item_name: "{{ template_name }}"
      register: __res

    - name: Assert values
      ansible.builtin.assert:
        that:
          - (__res.library_item_info | length) == 0
```

File: tests/integration/targets/vmware_content_template/tasks/main.yml [hunk 2]
```diff
@@ -60,6 +60,20 @@
...
+    - name: Delete template from content library
+      vmware.vmware.content_template:
+        validate_certs: false
+        hostname: "{{ vcenter_hostname }}"
+        username: "{{ vcenter_username }}"
+        password: "{{ vcenter_password }}"
+        port: "{{ vcenter_port }}"
+        template: "{{ template_name }}"
+        library: "{{ library }}"
+        vm_name: "{{ vm }}"
```
Comment (anna-savina) [inline tests/integration/targets/vmware_content_template/tasks/main.yml:72]: vm_name is a mandatory variable for 'absent' state? It seems wrong to me.

Comment (mikemorency) [reply tests/integration/targets/vmware_content_template/tasks/main.yml:72]: created a ticket in jira for the next release :+1:

**Review by anna-savina — APPROVED** [review]
LGTM

**Comment by prabinovRedhat** [comment]
lgtm

**Review by prabinovRedhat — APPROVED** [review]

**Pull request merged**